cargo-concordium/src/build.rs

```diff
@@ -1141,18 +1137,16 @@ pub(crate) fn build_and_run_integration_tests(
     // when allowing debug output, we make sure that test output is not captured.
     if allow_debug {
         // check if the user has already supplied extra test flags
-        let (test_args, show_output) =
```

Comment:
A for-loop is always more readable than a fold in my opinion 🤷 

Also a bit shorter